codefresh.yml

```diff
@@ -0,0 +1,30 @@
+version: '1.0'
+stages:
+  - clone
+  - test  
+steps:
+  main_clone:
+    title: Clone repository
+    stage: clone
+    type: git-clone
+    repo: ${{CF_REPO_OWNER}}/${{CF_REPO_NAME}}
+    revision: ${{CF_SHORT_REVISION}}
+    git: github
+  setup_github_token:
+    title: Setup GitHub token
+    stage: clone
+    image: codefresh/cli
+    commands:
+      - cf_export GITHUB_TOKEN=$(codefresh get context github --decrypt -o yaml | yq -y .spec.data.auth.password)
+  tests:
+    stage: test        
+    title: Run code quality
+    image: gcr.io/linio-support/docker-php:7.4
+    commands:      
+      - test -n "${GITHUB_TOKEN}" && composer config -g github-oauth.github.com ${GITHUB_TOKEN}
+      - composer install --no-scripts -o --no-interaction --no-suggest
+      - composer lint:check
+      - composer phpunit
+      - composer phpstan
+    environment:
+      - GITHUB_TOKEN=${{GITHUB_TOKEN}}
```

Comment:
Don't double bracket the `GITHUB_TOKEN`. This will cause the token to be output in plain text.